tool/conda/singa/build.sh

```diff
@@ -31,9 +31,16 @@ else
 	USE_CUDA=ON
 fi
 
+
+if [ $DIST == "ON" ]; then
+	USE_DIST=ON
+else
+	USE_DIST=OFF
+fi
+
 mkdir build
 cd build
 cmake -DCMAKE_INSTALL_PREFIX=$PREFIX -DUSE_CUDA=$USE_CUDA \
-	-DUSE_PYTHON3=ON -DUSE_MKLDNN=ON -DCMAKE_OSX_SYSROOT=${CONDA_BUILD_SYSROOT} ..
+	-DUSE_PYTHON3=ON -DUSE_DNNL=OFF -DUSE_DIST=$USE_DIST -DCMAKE_OSX_SYSROOT=${CONDA_BUILD_SYSROOT} ..
```

Comment:
USE_DNNL is a variable whose value depends on the environment. It should not be hard coded.

Reply:
DNNL is necessary for CNN, so will change to -DUSE_DNNL=ON after DNNL package is ready. (In the past -DUSE_MKLDNN=ON at default)